lib/cdo/delete_accounts_helper.rb

```diff
@@ -186,6 +186,12 @@ def remove_census_submissions(email)
     Census::CensusSubmission.where(submitter_email_address: email).each(&:destroy)
   end
 
+  # Removes EmailPreference records associated with this email address.
+  # @param [String] email An email address
+  def remove_email_preferences(email)
+    EmailPreference.where(email: email).each(&:destroy)
```

Comment:
EmailPreference has a unique constraint on `email` so maybe use `find_by(email: email)`?

Reply:
I assume the email being passed as an argument by the caller came from the User model and has already been stripped and lower-cased?

Reply:
Hmm, good question.  I'll have to investigate.  I know we strip and downcase before we hash emails, but we might store the original formatting for display back to the user.

Reply:
Looks like User model strips and downcases before save:
https://github.com/code-dot-org/code-dot-org/blob/12f8a844cd81d85c00512c9c5e8c1f5f89c6dc21/dashboard/app/models/user.rb#L544-L562